### DISCUSSION TITLE
Cannot bind Jquery Country select(countrySelect.js) plugin with Livewire model

### DISCUSSION THREAD

**Discussion by maaazzz** [livewire/livewire#2513] [1. Help]
###Cannot bind data with jquery country plugin(countrySelect.js). I want to update user data which includes country as well. but when I update the data it returns null. here is my code

**blade**
`<input id="country_selector" type="text" wire:model="country">`

**Livewire**
```
 public $user,$first_name,$last_name,$email,$country,$phone;
 public function updateDetails($id)
    {
dd($this->country);
        $user = User::findOrFail($id);
        $user->first_name = $this->first_name;
        $user->last_name = $this->last_name;
        $user->email = $this->email;
        $user->phone = $this->phone;
        $user->country = $this->country;
        $user->save();
        session()->flash('success','Profile Updated Successfully');
    }
```
**Context**
- Livewire version: [e.g. 2.2.3]
- Laravel version: [e.g. 8.3.1]
- Alpine version: [e.g. 2.3.0]
- Browser: [e.g. Chrome, Safari]

**Comment by austenc** [comment]
@maaazzz 

I suspect that the jquery plugin is manipulating the DOM and it's throwing Livewire off. You may find some success with using `wire:ignore`. See this page for an example (you'll have to scroll down a bit, but I suggest reading all of it anyway) https://laravel-livewire.com/docs/2.x/alpine-js#creating-a-datepicker.
